docs_devel/docs/30.CodingStyles.md

```diff
@@ -36,11 +46,11 @@ You can check your code with;
 You can check your code with Checkstyle linting tool.
 A project configuration is stored in `config/checkstyle/checkstyle.xml`
 We treat checkstyle lint error as warning, but you are recommended to
-improve your code before sending your patch to core team.
+improve your code before sending your patch to a core team.
```

Comment:
to *the* core team ?

Reply:
I agree, there is actually only one core team unless we later decide to delegate review depending on features or modules